lib/smart_answer_flows/marriage-abroad/outcomes/_services_and_fees.govspeak.erb

```diff
@@ -32,9 +32,9 @@
     <%= render partial: 'link_to_consular_fees.govspeak.erb',
                locals: { calculator: calculator } %>
   <% end %>
+<% end %>
 
-  <% if calculator.services.any? && calculator.services_payment_partial_name.present? %>
-    <% partial = 'payment_information_partials/' + calculator.services_payment_partial_name + '.govspeak.erb' %>
-    <%= render partial: partial, locals: { calculator: calculator } %>
-  <% end %>
+<% if calculator.services.any? && calculator.services_payment_partial_name.present? %>
+  <% partial = 'payment_information_partials/' + calculator.services_payment_partial_name + '.govspeak.erb' %>
+  <%= render partial: partial, locals: { calculator: calculator } %>
```

Comment:
You might well be right that "countries with consular facilities" is the correct concept to use. Unfortunately, it's not obvious to me that we'd be able to get a definitive answer about that. At least not in the short term.

In an ideal world, I think we'd get together with the content team/department and try to understand the fundamental rules underpinning marriage-abroad. In the hope that we could then improve the code.

In the meantime; given that the most recent content change requests seem to be based on individual countries rather than, for example, "countries with consular facilities", I'm relatively comfortable with moving more data to marriage_abroad_services.yml. This might be at the cost of losing some higher level concepts, but I think that's probably OK in the short term if it means it's easier to make changes.